qase-python-commons/src/qase/commons/logger.py

```diff
@@ -27,6 +27,6 @@ def log_debug(self, message: str):
             self.log(message, 'debug')
 
     @staticmethod
-    def _get_timestamp(format: str = "%Y%m%d_%H:%M:%S"):
+    def _get_timestamp(format: str = "%Y%m%d"):
```

Comment:
Can we leave the hour-minute-second part, but keep it in a Windows-compatible format? The exact time is a valuable information.

```suggestion
    def _get_timestamp(format: str = "%Y%m%d_%H_%M_%S"):
```

Reply:
https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names#31976060

Reply:
Fixed